recipes/catalystcoop.pudl/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "catalystcoop.pudl" %}
+{% set version = "0.1.0a2" %}
```

Comment:
Why package an alpha release where there's 0.1.3?

Reply:
I'm guessing you're referring to [this package](https://pypi.org/project/pudl/), which sadly isn't ours. We have reached out to the developers, and they want to retain that name, even though the package hasn't been updated in a couple of years. We've been working on `catalystcoop.pudl` for about three years, and have users that have integrated it into their code code, and we have several thousand lines of our own, so using the `organization.package` format seemed easiest.

Reply:
More generally, as to why a pre-release -- we're trying to get the packaging and distribution details worked out in parallel with some final code changes in advance of a workshop next week where we want folks to be able to install the package easily, but the docs don't match up with the code right now, and we don't want to confuse people by pushing it out while it's still known to be incongruous.

Reply:
@isuruf Do you feel like the naming/versioning here is a blocking issue?